hap.rs

```diff
@@ -70,9 +70,23 @@ cpython::py_module_initializer!(hap, |py, m| {
         ctrlc_received.store(true, std::sync::atomic::Ordering::Relaxed)
     }).unwrap();
 
-    py.run(init_script, None, None).map(|_| PyNone);
+    // py.run(init_script, None, None).map(|_| PyNone);
+    let globals = PyDict::new(py);
+    py.run(init_script, Some(&globals), None)?;
 
-    eprintln!("Hap initialized!");
+    // Now passing `py` as the first argument to `get_item`
+    // Attempt to explicitly set symbolic_trace globally
+    if let Some(symbolic_trace) = globals.get_item(py, "symbolic_trace") {
+        py.run("global symbolic_trace", Some(&globals), None)?;
+    }
+
```

Comment:
what are the purpopses of these changes?

Reply:
These are just testing changes for debugging the module, doesn't effect the code, since we are using the hap module they gave us